source/includes/_role_categories.md

```diff
@@ -0,0 +1,266 @@
+# Role Categories
+## List Role Categories
+
+```javascript
+fetch('https://core.eventtia.com/v1/role_categories/<id>', {
+  method: 'GET',
+  headers: {
+    'Authorization': '<your token>',
+  }
+})
+```
+> Make sure you replace &lt;your token&gt; with the JWT you get when you authenticate. 
+
+> Example of a successful (200) response:
+
+```http
+HTTP/1.1 200 OK
+{
+  "data": [
+    {
+      "id": "4",
+      "type": "role_categories",
+      "attributes": {
+        "id": 4,
+        "name": "grupo 1"
+      }
+    }
+  ],
+  "links": {
+      "first": "http://localhost:3000/api/v1/es/v1/role_categories?page[number]=1",
+      "last": "http://localhost:3000/api/v1/es/v1/role_categories?page[number]=1",
+      "prev": null,
+      "next": null
+    }
+}
+
+```
+This endpoint list role_categories return it
```

Comment:
This endpoint allows you to list the role categories you have associated with your account.

Reply:
done